chatkit-core/src/integrationTest/kotlin/com/pusher/chatkit/ChatManagerSpek.kt

```diff
@@ -125,6 +125,18 @@ class ChatManagerSpek : Spek({
             assertThat(roomNames).containsExactly(GENERAL)
         }
 
+        it("loads user rooms with unread message counts") {
```

Comment:
I think we should add a test case for the case where the unread count has been updated i.e marking a message as read, receiving a `room_updated` event..